Gemfile

```diff
@@ -13,7 +13,7 @@ gem "middleman-minify-html"
 gem "middleman-autoprefixer"
 
 # Minification and optimization
-gem "middleman-imageoptim"
+# gem "middleman-imageoptim" # issues with middleman v4
```

Comment:
we should probably just remove the line if not going to use it. is there a compatibility update with this gem for the newer middleman?